test/test_mps.py

```diff
@@ -934,6 +934,110 @@ def leak_gpu0():
         with self.assertRaisesRegex(RuntimeError, r"MPS driver API confirmed .+"):
             leak_gpu0()
 
+
+# These tests were taken from test/test_nn.py
```

Comment:
We have OpInfo based tests for this. So this should already properly covered?

Reply:
I meant there is a regular OpInfo: https://github.com/pytorch/pytorch/blob/f542b31c9dea1937328d3c3ce03738ce340faf1e/torch/testing/_internal/common_methods_invocations.py#L17985

That one should already be ran by the existing tests in this file no?

Reply:
If these tests already run, I'll be happy to remove them. Is there a way to check it?

Reply:
In the CI logs on the mps machine, you can look for `nn_functional_pixel_shuffle`

Reply:
> In the CI logs on the mps machine, you can look for `nn_functional_pixel_shuffle`

Found these lines

```
  test_mps.py::TestConsistencyCPU::test_output_grad_match_nn_functional_pixel_shuffle_cpu_float16 PASSED [0.1346s] [ 21%]
  test_mps.py::TestConsistencyCPU::test_output_grad_match_nn_functional_pixel_shuffle_cpu_float32 PASSED [0.1138s] [ 21%]
```

Reply:
But `test_pixel_shuffle_unshuffle` that I copied here is not presented elsewhere.

Indeed, `test_pixel_shuffle_unshuffle` in `test_nn.py` does not use `device` inside. So, I can't see how it can be automatically checked for MPS

Reply:
Do these already cover this test you just copied here? (and the test in test_nn is just redundant)
Or they test different things?

Reply:
They're different

Reply:
IMO it's a bid redundant, but can be done as a followup PR

Reply:
@alexdremov can you please remove the test_nn copied test in this PR to avoid duplication? We are planning to enable test_nn testing with 'mps' device soon and will have this enabled there.